system-int-test/test/cmr/system_int_test/search/collection_keyword_search_test.clj

```diff
@@ -205,18 +209,32 @@
 
         ;; entry title
         "coll1" [coll1]
+        "\"coll1\"" [coll1]
+        "coll26" [coll26 coll26-1]
+         "\"coll26\"" [coll26 coll26-1]
         "Mitch made a (merry-go-round)" [coll2]
```

Comment:
Add a test case to search with combination of regular word and quotes, e.g. `Mitch "a (merry-go-round)"`

Reply:
I think this is a test case that we need to support.